### DISCUSSION TITLE
No Option for Account Settings

### DISCUSSION THREAD

**Discussion by rash522** [yhype/yhype-feedback#39] [Bug] [answered]
There is no option to edit my yhype account. I just want to edit my twitter account link, cz I have changed it. Is there any way to edit data in my profile?
![Screenshot 2024-12-03 131659](https://github.com/user-attachments/assets/9a7cbe15-b4ad-4ee6-8f56-8ffd36ae7202)

**Comment by antonkomarev** [comment, marked as answer]
@rash522 GitHub does not notify third party apps for the account changes, so we are just doing this manually once a day, your account will be updated today's night.

But I think it might be a feature, to run account sync manually.

**Reply by rash522** [reply]
@antonkomarev Thank you for the reply! yeah that's a cool feature and I also thought this might be a feature ;)
